ui/imports/utils/Global.qml

```diff
@@ -11,6 +11,7 @@ QtObject {
     property int settingsSubsection: Constants.settingsSubsection.profile
 
     property var userProfile
+    property bool appIsReady: false
```

Comment:
yeah :/ I removed it because it's definitely bad to store any state in a QML singleton; however I see it's a valid usecase here. What about we store this variable in NIM side? Because the way it's used here is gonna break half of the Storybook pages.

Any idea @micieslak ?

Reply:
@caybro yes, I am up for any idea you have. We need to define a clean line between context props which may be used before and those available after user logs in, but that requires a lot more changes.

If you want to deal with those warnings I can easily revert second commit from this PR (it doesn't affect my work here, just wanted to clean the log).

Reply:
Not a big issue, keep it, if it's just warnings :) Just make sure the storybook pages still work fine